instrumentation/opentelemetry-instrumentation-pymssql/test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+pymssql==2.3.2
```

Comment:
Can you do an explicit pip freeze like we do for [other instrumentations](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-mysql/test-requirements-0.txt) so users can just pip install the requirements file and have all libraries they need to test?

Reply:
Looking into it.

Reply:
Done!